tastyworks/models/order.py

```diff
@@ -97,8 +114,11 @@ def from_dict(cls, input_dict: dict):
         Parses an Order object from a dict.
         """
         details = OrderDetails(input_dict['underlying-symbol'])
+        details.order_id = input_dict['id'] if 'id' in input_dict else None
+        details.ticker = input_dict['underlying-symbol'] if 'underlying-symbol' in input_dict else None
```

Comment:
Can use a `.get()` here as well and the few below it. Would make this a lot cleaner (I see I made a _faux pas_ myself as well on line 119 :laughing: )